test/evergreen.yml

```diff
@@ -2187,14 +2187,12 @@ tasks:
       - func: "compile wiredtiger"
         vars:
           posix_configure_flags: --enable-strict ac_cv_func_ftruncate=no
-      - command: shell.exec
-        params:
-          working_dir: "wiredtiger/build_posix"
-          script: |
-            set -o errexit
-            set -o verbose
-            # ${test_env_vars|} $(pwd)/../test/csuite/random_abort/smoke.sh 2>&1
-            ${test_env_vars|} $(pwd)/test/csuite/test_truncated_log 2>&1
+      - func: "csuite smoke test"
+        vars:
+          test_name: random_abort
```

Comment:
The random_abort test was previously disabled in the task. Is it safe to re-enable it now?

Reply:
I think it is safe as this test has been around for a while. See its definition as part of the PR testing [here](https://github.com/wiredtiger/wiredtiger/blob/develop/test/evergreen.yml#L1121-L1134).

Reply:
If you think it should be added in a separate ticket, happy to create one and add a FIXME comment. I am suggesting this approach as the tickets that are being merged to `wt-8498-evergreen-improvements` are supposed to only improve the readability of the evergreen file and not change the number of tests being run.

Reply:
I'm happy with the current change. Thanks for fixing it up.